src/compiler/diagnosticMessages.json

```diff
@@ -2297,6 +2301,10 @@
         "category": "Error",
         "code": 2578
     },
+    "Cannot find name '{0}'. Do you need to change your target library? Try changing the `lib` compiler option to es2017 or later.": {
+        "category": "Error",
+        "code": 2579
+    },

```

Comment:
Can you remove the new messages and switch the old one to accept a 2nd template slot?